src/backend/app.js

```diff
@@ -20,12 +20,35 @@ app.use(express.json());
 app.use(cors());
 
 router.use("/meals", mealsRouter);
+app.get('/my-route', (req, res) => {
+  res.send("Hi friend");
+});
 
 if (process.env.API_PATH) {
   app.use(process.env.API_PATH, router);
 } else {
   throw "API_PATH is not set. Remember to set it in your .env file"
 }
+app.get("/future-meals", async (req, res) => {
+  const rows = await knex.raw("SELECT * FROM `meal` WHERE `when` > NOW()");
+  rows[0].length === 0 ? res.sendStatus(404) : res.send(rows.slice(0, 1))
```

Comment:
Even though there are no meals available, it shouldn't be an error response as it is responding with the correct data.
So the response is Status:500 and result:"There are not meals available".

Reply:
I will modified this ,thank you